src/main/java/com/eternalcode/parcellockers/ParcelLockers.java

```diff
@@ -132,6 +156,24 @@ private void softwareCheck() {
         logger.info("Your server is running on supported software, congratulations!");
         logger.info("Server version: " + this.getServer().getVersion());
     }
+
+    private boolean setupEconomy() {
+        if (this.getServer().getPluginManager().getPlugin("Vault") == null) {
+            return false;
+        }
+        
+        RegisteredServiceProvider<Economy> rsp = this.getServer().getServicesManager().getRegistration(Economy.class);
+        if (rsp == null) {
+            return false;
+        }
+        
+        this.economy = rsp.getProvider();
+        return true;
+    }
+
+    public Economy getEconomy() {
+        return this.economy;
+    }
```

Comment:
I don't like this method in main class.

Reply:
I'm sorry to hear that, we else can I move this method to?